### ISSUE TITLE
Create Filament Swatch Card

### ISSUE THREAD

**Issue by TUISTERa** [Donkie/Spoolman#420]
**Describe the solution you'd like**
A way to generate some kind of swatch card of every filament in the database . 

**Additional context**
](https://www.printables.com/model/937387-filament-swatch-card-with-label)

**Comment by T3chGuy77** [comment]
I don't think this is a good feature request. I have a color swatch gcode file for each type of filament ready to go on my printers. I just load the new filament and print the correct one for PLA, TPU, ABS, PETG etc. I get what you're asking but from a dev viewpoint I doubt it will happen.

**Comment by mikeosh** [comment]
This duplicates what The Print Codex does....  Buy or print your own.

https://theprintcodex.com/

<https://theprintcodex.com/>
***@***.***_699540/models
<https://theprintcodex.com/>

**Comment by akira69** [comment]
> I don't think this is a good feature request. I have a color swatch gcode file for each type of filament ready to go on my printers. I just load the new filament and print the correct one for PLA, TPU, ABS, PETG etc. I get what you're asking but from a dev viewpoint I doubt it will happen.

Why not?
it seems pretty easy on the surface - just like the spool print page, have a filament print page

as @mikeosh mentions, you can buy swatches... but who wants to buy swatches?  print your own and print a generic filament label to go on it.  Or buy the swatches and print filament labels that interface with spoolman database.  it opens the filament page, you click "show spools" and either see you have one, or realize you need to create one, or determine you need to buy it.

3dfilamentprofiles.com supports filament labels see #824

**Comment by akira69** [comment]
**#846 Filament Label Printing
